src/screens/Learning.js

```diff
@@ -133,7 +151,11 @@ export default ({
           </View>
           <View style={styles.heading}>
             <SectionHeading text="Category: " />
-            <Text>{currentCategoryName}</Text>
+            <Text>
+              {openSeenPhrases
```

Comment:
replace this with a check if current category is ##seen-phrases##.

You can set it in home `openSeenPhrases`

Reply:
remove openSeenPhrases and setOpenSeenPhrases from redux etc